app.R

```diff
@@ -8,26 +8,68 @@ sapply(list.files("R", full.names = TRUE, recursive = TRUE), source, .GlobalEnv)
 
 config <- yaml::read_yaml("./mainDashboard.yaml")
 
-confirmed <- usa_facts_data.confirmed() %>% 
-  tidyr::gather(Date, confirmed, -c(countyFIPS, `County Name`, State, stateFIPS))
+county <- create_data.county()
```

Comment:
Running each of the `create_data` functions outputs the following warning:

Warning message:
In write.table(df, here::here("./data/usafacts/", csv_name), col.names = NA,  :
  invalid char string in output conversion

Not sure if this matters because we are not writing CSVs?

Reply:
Hmm, I wasn't getting that message. Do you see if it wrote the CSVs out? It's not too important, but I would also like to fix it if it's not working.

Reply:
It's not outputting any CSVs into my data folders, unfortunately